samples/04_gis_analysts_data_scientists/tabular_data_supervised_learning_using_automl.ipynb

```diff
@@ -0,0 +1,1201 @@
+{
```

Comment:
<p>This path with break on user's end and also on AGS Notebooks. Can you replace the it something that is relative to this notebook?</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/977/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>

Reply:
<p>Done</p>